clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -2,13 +2,14 @@ import { h, FunctionComponent } from "preact";
 import { ButtonType, ExperienceConfig } from "../lib/consent-types";
 import Button from "./Button";
 import { useHasMounted } from "../lib/hooks";
+import CloseButton from "./CloseButton";
 
 interface BannerProps {
   experience: ExperienceConfig;
   onAcceptAll: () => void;
   onRejectAll: () => void;
   onManagePreferences: () => void;
-  managePreferencesLabel?: string;
+  onClose: () => void;
```

Comment:
I had just suggested to @eastandwestwind that we could get away with not passing both the `bannerIsOpen` and a setter, but it turns out we do need the setter here, or at least a closer now that the components have close buttons 🥲

Reply:
oh this makes sense, thanks!